README.md

```diff
@@ -1,18 +1,155 @@
-# ECC Python Cookiecutter Template
+# Python Cookiecutter Template
 
 This is a Cookiecutter template for creating a Python project with Poetry,
 including configurations for various project settings.
 
 ## Usage
 
-1. **Install Cookiecutter** (if you haven't already):
+1. **Install cruft** (if you haven't already):
 
 ```sh
-pip install cookiecutter
+pip install cruft
 ```
 
 2. **Generate a New Project**:
 
 ```sh
-cookiecutter gh:elixir-cloud-aai/cookiecutter-python
+cruft create https://github.com/elixir-cloud-aai/cookiecutter-python.git
 ```
+
+## Configurations
+
+### Package manager - Poetry
+
+This project uses [Poetry](https://python-poetry.org/) as a package manager.
```

Comment:
Please consistently use reference style links of the form "[text][ref-name]", with "[ref-name]: <URL>" at the bottom of the page (sorted alphabetically)